docs/references/configuration.rst

```diff
@@ -599,11 +599,17 @@ jwt-aud
   **In-Database** pgrst.jwt_aud
   =============== =================================
 
-  Specifies the `JWT audience claim <https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3>`_. If this claim is present in the client provided JWT then you must set this to the same value as in the JWT, otherwise verifying the JWT will fail.
+  Specifies the `JWT audience claim <https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3>`_.
+
+  By default, PostgREST identifies with all audiences and allows the JWT for any ``aud`` claim.
+
+  Once this config is set, PostgREST will compare this value to the one in the JWT ``aud`` claim.
+  If this comparison fails, then the token is rejected.
```

Comment:
This doesn't mention the case of aud arrays (` "aud": ["notyouraudience", "youraudience", "anotheraudience"]`), where we'll look for one matching element. I think that should be mentioned as it's "our interpretation"?